packages/components-native/src/InputText/InputText.tsx

```diff
@@ -65,7 +65,7 @@ export interface InputTextProps {
   /**
    * Determines what keyboard is shown
    */
-  keyboard?:
+  readonly keyboard?:
```

Comment:
Any concern with making this `readonly`? it was a warning lint that auto fixed onsave/oncommit

Reply:
All our props are now auto-linted to readonly, so no worries here nope.